homeassistant/components/cast/media_player.py

```diff
@@ -263,13 +265,19 @@ async def _async_cast_discovered(self, discover: ChromecastInfo):
         _LOGGER.debug("Discovered chromecast with same UUID: %s", discover)
         self._cast_info = discover
 
-    async def _async_cast_removed(self, discover: ChromecastInfo):
+    async def _async_cast_removed(self, discover: ChromecastInfo) -> None:
         """Handle removal of Chromecast."""
 
-    async def _async_stop(self, event):
+    async def _async_stop(self, event: Event) -> None:
         """Disconnect socket on Home Assistant stop."""
         await self._async_disconnect()
 
+    def _get_chromecast(self) -> pychromecast.Chromecast:
+        """Ensure chromecast is available."""
+        if self._chromecast is None:
+            raise HomeAssistantError("Chromecast is not available.")
+        return self._chromecast
+
```

Comment:
I have added this function to avoid using `if` or `assert` throughout the code.
But maybe a code-owner has a better idea?

Reply:
I think this is fine 👍